volontulo/tests/views/test_auth.py

```diff
@@ -67,3 +69,57 @@ def test_successful_registration(self):
         )
         self.assertIn('_auth_user_id', self.client.session)
         self.assertEqual(User.objects.all().count(), 1)
+
+
+class TestAuth(TestCase):
+    u"""Class responsible for testing users authentication view."""
+
+    @classmethod
+    def setUpTestData(cls):
+        # volunteer user
+        TestUsersProfile.initialize_empty_volunteer()
+
+    def setUp(self):
+        u"""Set up each test."""
+        self.client = Client()
+
+    # pylint: disable=invalid-name
+    def test__logged_out_anonymous_user(self):
+        u"""Testing logout view for anonymous user"""
+        response = self.client.get('/logout', follow=True)
+
+        self.assertRedirects(
+            response,
+            'http://testserver/login?next=/logout',
```

Comment:
Maybe it will work like that, but we should change that behavior, because immediately rendering logout after login will  create situation, that user will never successfully login.